docs/tutorials/04_ParticleFilter.py

```diff
@@ -144,10 +144,26 @@
 # many resampling schemes, and almost as many choices as to when to undertake resampling. The
 # systematic resampler is described in [#]_, and in what follows below resampling is undertaken
 # at each time-step.
+
+# %%
+# Use of Effective Sample Size resampler (ESS)
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# Resampling removes particles with a low weight and multiplies particles with a high weight.
```

Comment:
perhaps multiplies -> duplicates to distinguish the difference between "multiplication" and "copying".